src/server/kube-store.js

```diff
@@ -128,7 +138,89 @@ class KubeStore {
                 return;
             }
             default: {
-                throw new Error(`Could not patch noobaa, got ${JSON.stringify(body)}`);
+                throw new Error(`Could not patch noobaa, (status code: ${status_code}) got ${JSON.stringify(body)}`);
+            }
+        }
+    }
+
+    async create_backingstore(new_store) {
+        await this._init();
+        const path = get_backingstores_path(this._k8s_namespace);
+        const { status_code, body } = await this._make_k8s_api_request('POST', path, new_store);
+        switch (status_code) {
+            case 201: {
+                return body;
+            }
+            case 404: {
```

Comment:
I don't think you can a 404 for a POST operation, also why do you need to return the body in the case of create operation? ( the 201)